server/routes/userRoutes.js

```diff
@@ -1,14 +1,14 @@
-import express from "express";
-import userController from "../controllers/userControllers";
-import Users from "../model/Users";
-import validation from "../validation/userValidation";
+import express from 'express';
+import userController from '../controllers/userControllers';
+import Users from '../model/Users';
+import validation from '../validation/userValidation';
 
 const userRoute = express.Router();
 // get all users
-userRoute.get('/users',(req, res)=>{
-    return res.status(200).json({ users: Users });
-})
+userRoute.get('/users', (req, res) => res.status(200).json({ users: Users }));
```

Comment:
@mic7x4 can you think about moving this controller to users' controller file?

Reply:
yeah